src/utils/cloning.ts

```diff
@@ -157,6 +159,19 @@ function cloneASTNodeImpl<T extends ASTNode>(
       node.referencedDeclaration,
       node.raw,
     );
+  } else if (node instanceof TupleExpression) {
+    const tupleComponents = node.vOriginalComponents.map((component) => {
+      return component !== null ? cloneASTNodeImpl(component, ast, remappedIds) : null;
+    });
+    newNode = new TupleExpression(
+      replaceId(node.id, ast, remappedIds),
+      node.src,
+      'TupleExpression',
+      node.typeString,
+      node.isInlineArray,
+      tupleComponents,
```

Comment:
The non-null elements need to be cloned with `cloneASTNodeImpl`

Reply:
Looks like the none-null elements are being cloned with that function, are you talking about the null ones?

Reply:
Right, sorry, my default check is to make sure that there's no ASTNodes that get passed to the constructor without being wrapper in cloneASTNodeImpl. I managed to completely miss that they're being cloned further up, this is all good